app/views/spree/admin/shared/_stimulus_sortable_header.html.haml

```diff
@@ -1,5 +1,5 @@
 %th
-  %a{ "data-action": "click->search#changeSorting", "data-column": "#{column}", "data-current": sorted.to_s }
+  %a{ "data-controller": "search", "data-action": "click->search#changeSorting", "data-column": "#{column}", "data-current": (sorted || default).to_s }
```

Comment:
It's better to associate the `search `controller here because it doesn't have to depend upon the parent to use it.